multicluster/config/rbac/kustomization.yaml

```diff
@@ -7,8 +7,10 @@ resources:
 - service_account.yaml
 - role.yaml
 - role_binding.yaml
-- leader_election_role.yaml
-- leader_election_role_binding.yaml
+# We disabled leader election in manager, so this role and
```

Comment:
What does "manager" refer to here?

Reply:
I mean an option in a kubebuilder manager initialization which is used by leader and member controllers.  let me change it to controller for easy understanding.